tests/beta/test_default_tensor_tf.py

```diff
@@ -774,7 +774,7 @@ def test_error_classical_diff_autograd(self, tol):
         """Tests that an error is raised if for the classical differentiation
         method when using the autograd interface"""
         interface = "autograd"
-        diff_method = "classical"
+        diff_method = "backprop"
```

Comment:
Two more occurrences on lines [770](https://github.com/XanaduAI/pennylane/blob/178b772435e1f22e19afc8aa828d9221d20ad719/tests/beta/test_default_tensor_tf.py#L770), [792](https://github.com/XanaduAI/pennylane/blob/178b772435e1f22e19afc8aa828d9221d20ad719/tests/beta/test_default_tensor_tf.py#L792) too at the end of the error message

Reply:
Good catch my `sed` skills failed me!

Reply:
Oh you know why? I only searched for `"classical"`, not `'classical'` 🤦‍♂️

Reply:
weird, I don't see this

Reply:
ahh, I see it now, nevermind :facepalm: